composer.json

```diff
@@ -31,29 +35,31 @@
     }
   },
   "require": {
-    "php": "^7.1.3",
-    "illuminate/cache": "^5.8",
-    "illuminate/config": "^5.8",
-    "illuminate/console": "^5.8",
-    "illuminate/container": "^5.8",
-    "illuminate/contracts": "^5.8",
-    "illuminate/database": "^5.8",
-    "illuminate/events": "^5.8",
-    "illuminate/filesystem": "^5.8",
-    "illuminate/log": "^5.8",
-    "illuminate/support": "^5.8",
-    "illuminate/view": "^5.8",
+    "php": "^7.2",
+    "illuminate/cache": "^6.3",
+    "illuminate/config": "^6.3",
+    "illuminate/console": "^6.3",
+    "illuminate/container": "^6.3",
+    "illuminate/contracts": "^6.3",
+    "illuminate/events": "^6.3",
+    "illuminate/filesystem": "^6.3",
+    "illuminate/log": "^6.3",
+    "illuminate/support": "^6.3",
+    "illuminate/view": "^6.3",
     "league/flysystem": "^1.0",
     "roots/support": "dev-master",
     "symfony/var-dumper": "^4.3"
   },
   "require-dev": {
     "filp/whoops": "^2.5",
     "mikey179/vfsstream": "^1.6",
-    "phpunit/phpunit": "^7.5|^8.0",
+    "phpunit/phpunit": "^8.0",
     "roave/security-advisories": "dev-master",
     "squizlabs/php_codesniffer": "^3.3"
   },
+  "suggest": {
+    "filp/whoops": "Required for friendly error pages in development (^2.5)."
```

Comment:
`filp/whoops` is already required for dev, right? This suggestion will never be printed... 😉 
Oh and Laravel 6+ use https://github.com/facade/ignition, maybe a switch here too?

Reply:
filp/whoops will get printed when requiring Acorn in projects (e.g. Sage) and is required in those to initialize (a PR will be put out for Sage once this is merged).

Also we are looking at Ignition, but it will take some time before it is implemented (I have to rewrite most of the implementation as it hardcodes `illuminate/foundation`)